CHANGELOG

```diff
@@ -13,7 +13,7 @@ liburing-2.2 release
 - Add support for registered ring fds, io_uring_register_ring_fd(),
   reducing the overhead of an io_uring_enter() system call.
 - Add support for the message ring opcode.
-- Add support for newer request cancelation features.
+- Add support for newer request cancellation features.
```

Comment:
These aren't typos, both are valid spellings.

Reply:
Ah, thanks, thought I had confirmed this after a "dict cancelation" returning nothing, but I see this is a common spelling in the US. Will drop, and file a report against lintian to not warn on these.

Reply:
It is a US vs UK thing, both are valid spellings in the US. UK may only using the double ll variant, not sure.